modules/led.sh

```diff
@@ -91,6 +94,27 @@ function dance {
   led green "$current_green"
 }
 
+function christmas {
+  current_red=$(led "red")
+  current_green=$(led "green")
+
+  led green none
+  led red none
+  sleep 1
+  
+  led red timer
+  led green timer
+  sleep 8
+
+  led green none
+  led red none
+  sleep 1
+
+  led red "$current_red"
+  led green "$current_green"
+
+}
+
 function led_help {
   echo ""
   echo "Usage: $(basename "$0") led [green|red|dance] [mode]"
```

Comment:
Maybe add christmas to help?
`echo "Usage: $(basename "$0") led [green|red|dance|christmas] [mode]"`